buildpack_integration_test.go

```diff
@@ -16,8 +20,11 @@ import (
 	"github.com/paketo-buildpacks/occam"
 	. "github.com/paketo-buildpacks/occam/matchers"
 	"github.com/paketo-buildpacks/packit/v2/pexec"
+	"github.com/paketo-buildpacks/packit/v2/vacation"
 )
 
+const REGISTRY_IMAGE = "registry:2"
```

Comment:
could you explain a bit about the test changes you made here to get things passing?

Reply:
This is what I mentioned in the commit comment.

>- Modify integration tests so that we don't use skopeo, which can't handle multi-arch images
>
>The latter was a little tricky. I couldn't find a way to load the multi-arch OCI TAR file into the Docker daemon so we could run the test. Instead, I had to spin up a Registry running in Docker, then load the multi-arch OCI TAR files for the stack into the internal Registry. Then we can build from the registry.

It was failing because `skopeo` doesn't work with multi-arch image. At least, it doesn't work with multi-arch images the way we are generating them.

The integration test is building a stack image & running a build with it. The critical step is taking the TAR file from the build step and loading it into the Docker daemon so the build can be run. However, we can't do that with `skopeo`.

I looked around to see if I could make the go container library load an OCI TAR file into the Daemon but it doesn't seem to be supported. It can load in a Docker TAR file, but that is not the same.

I was thinking about trying to convert them or making `jam` write out a Docker TAR file, but it ended up being the easiest way to run a local registry in Docker. The go container library can quickly load the OCI TAR into the local registry, and the build can then run by pointing to the local registry for the stack image.

So it adds a little overhead, but it was not a significant amount of additional time (loading to the Daemon vs loading to a local registry).

If the local registry is a problem, we could load the image to a remote registry. I avoided that option for two reasons: a.) I was worried about network overhead and b.) it would mean we need to publish somewhere semi-public. Let me know what you think.

Reply:
Thank you very much for the explanation - that clears things up for me. I think this approach makes sense over pushing to an intermediate remote testing registry